test/parallel/test-domain-multiple-errors.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
What is the difference between this test and `test/parallel/test-domain-error-types.js`?